### PR TITLE
[#189 + #193] Games page update

### PR DESCRIPTION
**Pull request by Wolfboy248**

### CODE REVIEW

File: frontend/src/components/pages/gamecat.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Routes, useNavigate } from 'react-router-dom';
+
+export default function GameCategory(prop) {
+    const [iteration, setIteration] = React.useState(prop.iteration);
+    const [category, setCategory] = React.useState(prop.cat);
+    const [gameinfo, setGameinfo] = React.useState(prop.gameinfo);
+
+    let cats = [
```
Comment (pektezol) [inline frontend/src/components/pages/gamecat.js:10]: Can't you get the category ids from the API to set them dynamically?

File: frontend/src/components/pages/map.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Routes, useNavigate } from 'react-router-dom';
+
+export default function MapItem(prop) {
+    return (
+        <div>Hello from MapItem</div>
```
Comment (pektezol) [inline frontend/src/components/pages/map.js:7]: Is this a debug page?

File: frontend/src/components/pages/gamecat.js (hunk 1 above)
Comment (Wolfboy248) [reply frontend/src/components/pages/gamecat.js:10]: i tried, and no, not with the way the ids are set up. It's easier just to set it manually

File: frontend/src/components/pages/map.js (hunk 2 above)
Comment (Wolfboy248) [reply frontend/src/components/pages/map.js:7]: That was a mistake, file's not supposed to be there

**Pull request closed without merging**